### ISSUE TITLE
"Open Visual Editor" doesn't work on machines created inside a function

### ISSUE THREAD

**Issue by hugobdev** [statelyai/editor-feedback#32]
Given

```js
function createSomethingMachine () {
    return createMachine({
        id: 'something'
    })
}
```

I expected "Open Visual Editor" to work. When I click it, I receive a warning in VS Code: "Could not find a machine at the current cursor".

I'm using `xstate@4.29.0` and `XState VSCode v1.5.6`. "Open Inspector" works as expected. "Open Visual Editor" also works when a machine is created in the top-level of a module.

**Comment by hugobdev** [comment]
@mattpocock that did the trick. Thank you!

**Comment by mattpocock** [comment]
Fixed in 1.5.10
